brokerapi/brokers/bigtable/broker.go

```diff
@@ -127,21 +126,14 @@ func (b *BigTableBroker) Provision(instanceId string, details brokerapi.Provisio
 }
 
 // deletes the instance associated with the given instanceID string
```

Comment:
This function's header doc doesn't conform to the Godoc format: https://blog.golang.org/godoc-documenting-go-code